### PR TITLE
test: update cypress fixture file uploads to handle pdf and other files

### PR DESCRIPTION
**Pull request by frank-dodion**
## Ticket Number
AS-1577

## Description of change
Update cypress fixture file uploads to handle pdf and other files

### CODE REVIEW

File: e2e-tests/cypress/integration/as-1577.feature [hunk 1]
```diff
@@ -0,0 +1,7 @@
+@wip
+Feature: Test pdf file uploads
+
+  Scenario: Upload appeal with range of other documents
+    Given  an appellant has prepared an appeal for as-1577
```
Comment (swdpcomputing) [inline e2e-tests/cypress/integration/as-1577.feature:5]: Double space

File: e2e-tests/cypress/integration/common/as-1577.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
+import { Given, When, Then } from 'cypress-cucumber-preprocessor/steps';
+
+Given('an appellant has prepared an appeal for as-1577', () => {
```
Comment (swdpcomputing) [inline e2e-tests/cypress/integration/common/as-1577.js:3]: Just a question Frank, I've not seen such a long statement, is this simply checking a full application process?

File: e2e-tests/cypress/integration/common/as-1577.js [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+  cy.answerOwnsTheWholeAppeal();
+  cy.clickSaveAndContinue();
+
+  cy.answerCanSeeTheWholeAppeal();
+  cy.clickSaveAndContinue();
+
+  cy.answerSiteHasNoIssues();
+  cy.clickSaveAndContinue();
+
```
Comment (swdpcomputing) [inline e2e-tests/cypress/integration/common/as-1577.js:79]: Blank line

File: e2e-tests/cypress/support/appeal-statement-submission/uploadAppealStatementFile.js [hunk 4]
```diff
@@ -1,6 +1,6 @@
 import 'cypress-file-upload';
 
 module.exports = (path) => {
-  cy.get('#appeal-upload').attachFile(path);
+  cy.get('#appeal-upload').attachFile({filePath: path, encoding: 'binary' });
```
Comment (swdpcomputing) [inline e2e-tests/cypress/support/appeal-statement-submission/uploadAppealStatementFile.js:4]: Space before filePath

File: e2e-tests/cypress/support/appellant-submission-decision-letter/uploadDecisionLetterFile.js [hunk 5]
```diff
@@ -1,6 +1,6 @@
 import 'cypress-file-upload';
 
 module.exports = (path) => {
-  cy.get('#decision-upload').attachFile(path);
+  cy.get('#decision-upload').attachFile({filePath: path, encoding: 'binary' });
```
Comment (swdpcomputing) [inline e2e-tests/cypress/support/appellant-submission-decision-letter/uploadDecisionLetterFile.js:4]: Space before filePath

File: e2e-tests/cypress/support/appellant-submission-supporting-documents/uploadSupportingDocuments.js [hunk 6]
```diff
@@ -7,7 +7,7 @@ module.exports = (paths) => {
     paths = [paths];
   }
 
-  paths.forEach((path) => documentUpload.attachFile(path));
+  paths.forEach((path) => documentUpload.attachFile({filePath: path, encoding: 'binary' }));
```
Comment (swdpcomputing) [inline e2e-tests/cypress/support/appellant-submission-supporting-documents/uploadSupportingDocuments.js:10]: Space before filePath

File: e2e-tests/cypress/support/appellant-submission-upload-application/uploadPlanningApplicationFile.js [hunk 7]
```diff
@@ -1,6 +1,6 @@
 import 'cypress-file-upload';
 
 module.exports = (path) => {
-  cy.get('#application-upload').attachFile(path);
+  cy.get('#application-upload').attachFile({filePath: path, encoding: 'binary' });
```
Comment (swdpcomputing) [inline e2e-tests/cypress/support/appellant-submission-upload-application/uploadPlanningApplicationFile.js:4]: Space before filePath

**Review by swdpcomputing — COMMENTED** [review]
Just a few formatting suggestions and a query.

**Review by swdpcomputing — APPROVED** [review]
LGTM

**Review by danWatkinson — DISMISSED** [review]
yassine is now nothing to me. frank you are my new hero.

**Review by danWatkinson — APPROVED** [review]
❤️

**Pull request merged**